ui/components/app/confirm/info/row/alert-row/alert-row.tsx

```diff
@@ -58,15 +69,23 @@ export const AlertRow = (props: AlertRowProps) => {
   const inlineAlert = hasFieldAlert ? (
     <Box marginLeft={1}>
       <InlineAlert
-        onClick={() => {
-          // intentionally empty
-        }}
+        onClick={handleOpenModal}
         severity={getSeverityAlerts(variant)}
       />
     </Box>
   ) : null;
 
   return (
-    <ConfirmInfoRow {...confirmInfoRowProps} labelChildren={inlineAlert} />
+    <>
+      {alertModalVisible && (
+        <MultipleAlertModal
+          alertKey={alertKey}
+          ownerId={ownerId}
+          onFinalAcknowledgeClick={handleCloseModal}
+          onClose={handleCloseModal}
+        />
+      )}
+      <ConfirmInfoRow {...confirmInfoRowProps} labelChildren={inlineAlert} />
```

Comment:
If the `style` and `variant` are row properties, do we need to pass those also?

Reply:
I need to change the `style`  to add the `background: 'transparent'` and depending on the `variant` change the `color` as well.

Reply:
We can still use them, but if they are defined in the core `Row` component, do we need to pass them to it also?

Reply:
Sorry, I was confused because I included the  `style` in `confirmInfoRowProps` but missed the `variant`.  Thanks to your comment I also realised that `variant` could be `undefined` and the severity in the alert should dictate the colors.